src/web/src/utils.js

```diff
@@ -294,3 +301,28 @@ export const exportScheduleToIcs = (selectedCourses) => {
     })}_Schedule`
   );
 };
+
+/**
+ * Export The Course Schedule To An Image Format, Probably PNG.
+ * Can Come Back Here + Add More File Format Options, Such As JPEG, ...
+ */
+export const exportScheduleToImage = (selectedCourses, currentSemester) => {
+  // Handle Special Case Where No Selected Courses On Schedule.
+  if(selectedCourses.length == 0){
+    alert("No Courses Found For Export To Image Data.")
+    return;
+  }
+  // Obtain Schedule Element Defined In Schedule.Vue File + Run Export To PNG. 
+  var currentScheduleNode = document.getElementById('allScheduleData')
+  domtoimage.toPng(currentScheduleNode, {bgcolor: currentScheduleNode.getAttribute('isDarkModeOn')})
+    .then(function (dataUrl) {
+        var link = document.createElement('a'); 
+        link.download = 'myYACS.N-' + currentSemester + '-Schedule.png';
```

Comment:
I don't think this needs to be as long a name, it confuses me when I download it, so it will probably confuse others. It's more important they see the semester than that they know its from yacs.n. I think even just "_semester__yacs-schedule.png" would be fine.

Reply:
Sounds good to me. 
Done.